src/engine/sparqlExpressions/PrefilterExpressionIndex.h

```diff
@@ -409,6 +409,11 @@ std::vector<PrefilterExprVariablePair> makePrefilterExpressionVec(
     const IdOrLocalVocabEntry& referenceValue, const Variable& variable,
     bool mirrored, bool prefilterDateByYear = false);
 
+//______________________________________________________________________________
+// Make a copy of the given <PrefilterExprVariablePair> vector.
```

Comment:
```suggestion
// Make a deep copy of the given <PrefilterExprVariablePair> vector.
// (They cannot be explicitly copied as they store unique_ptr)
```
But maybe even better:
Make the `PrefilterExprVariablePair` an explicit class with a proper copy constructor, because you use the unique_ptrness NOT for avoiding copies, but mostly for virtual type erasure).